### PR TITLE
Update ADOPTERS.md to add Spectro Cloud to the list

### PR DESCRIPTION
**Pull request by cloudmaniac**
**What this PR does / why we need it**: This PR adds Spectro Cloud as an adopter for KubeVirt. 

**Special notes for your reviewer**: None 🤗

### CODE REVIEW

**Comment by jean-edouard** [comment]
/release-note-none
/approve
/ok-to-test
Thank you @cloudmaniac !

File: ADOPTERS.md [hunk 1]
```diff
@@ -28,5 +28,6 @@
 | Vendor | Oracle | 2023 | [link](https://www.oracle.com) | As a distributor we are leveraging KubeVirt to enable VM workload on [Oracle Cloud Native Environment](https://www.oracle.com/linux/cloud-native-environment/), the Oracle multicloud and on-premises Kubernetes distribution. |
 | Vendor | PLATFORM9 | | [link](https://platform9.com/managed-kubevirt/) | Run Legacy and Cloud-Native Applications on Platform9 |
 | Vendor | Red Hat, Inc. | 2016 | [link](https://www.redhat.com) | As a distributor we are building OpenShift Virtualization on KubeVirt in order to enable VM workloads and -flows on Kubernetes. |
+| Vendor | Spectro Cloud | 2022 | [link]([https://www.spectrocloud.com/solutions/vms-on-kubernetes) | Spectro Cloud Palette's Virtual Machine Orchestrator (VMO) feature builds on KubeVirt. It gives enterprises an easy yet powerful way to bring their VM workloads into their Kubernetes clusters, on bare metal and at the edge — with full lifecycle management and unified policies and governance. |
```
Comment (awels) [inline ADOPTERS.md:31]: There is an extra `[` in the url part of the link here.

Comment (cloudmaniac) [reply ADOPTERS.md]: My bad! Link fixed.

**Comment by xpivarc** [comment]
Hi @cloudmaniac,
would you mind squashing your commits? Please make sure the commit/s are signed off. 
Great to see your use case!

**Comment by cloudmaniac** [comment]
@brianmcarey @xpivarc I managed to mess with my git history. 🙄
It will be easier if I just close this PR and open a new one with directly the correct link.

**Comment by cloudmaniac** [comment]
New [PR](https://github.com/kubevirt/kubevirt/pull/10729) created.

**Pull request closed without merging**